tests/runner/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.mule.tests</groupId>
+        <artifactId>mule-tests</artifactId>
+        <version>4.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>mule-tests-runner</artifactId>
+    <packaging>jar</packaging>
+    <name>Mule Test Runner</name>
+    <description>Test Runner for Mule Integration Tests</description>
+
+    <properties>
+        <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.aether</groupId>
+            <artifactId>aether-api</artifactId>
+            <version>${aetherVersion}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.aether</groupId>
+            <artifactId>aether-impl</artifactId>
+            <version>${aetherVersion}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule</groupId>
+            <artifactId>mule-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-extensions-support</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-artifact</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-container</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-deployment</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <!-- Override scope="test" from parent -->
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
+</project>
```

Comment:
Today I was checking a dependency that is used by the runner and not declared. @pablolagreca removed this transitive dependency and found this issue, as part of this PR it would be nice to add the missing declared dependency:
```
org.mule.tests:mule-tests-functional:jar:4.0-SNAPSHOT->org.mule.modules:mule-module-extensions-support:jar:4.0-SNAPSHOT:compile->org.mule.modules:mule-module-tooling-support:jar:4.0-SNAPSHOT:compile->org.mule.modules:mule-module-repository:jar:4.0-SNAPSHOT:compile->org.apache.maven:maven-aether-provider:jar:3.3.9:compile->org.apache.maven:maven-model:jar:3.3.9:compile
```
org.apache.maven:maven-model:jar:3.3.9 could be added as compile here.